operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -79,6 +80,8 @@
     public static final String CA_CRT = "ca.crt";
     public static final String IO_STRIMZI = "io.strimzi";
     public static final String ANNO_STRIMZI_IO_FORCE_RENEW = "strimzi.io/force-renew";
+    public static final String ANNO_STRIMZI_IO_CA_CERT_GENERATION = "strimzi.io/ca-cert-generation";
```

Comment:
Don't we need two annotations? One for the cluster ca and another for the clientsca?

Reply:
Yes, for now, the PR was just about cluster CA cert generation annotation but we can add the clients part as well if you prefer.